src/v4/component/CommunityStories/index.tsx

```diff
@@ -126,9 +126,9 @@ export default function CommunityStories({
       });
   }, [avatarUrl, communityId, displayName, hasStoryPermission, navigation]);
 
-  return (
-    <View style={styles.container}>
-      {loading ? (
+  const renderCommunityStory = useCallback(() => {
```

Comment:
we should not return components in `useCallback` because `useCallback` is a React Hook that lets you cache a function definition between re-renders. https://react.dev/reference/react/useCallback

Reply:
we can create a new component and passing props instead

Reply:
removed useCallback krub